user/postprocess/vtk.md

```diff
@@ -5,13 +5,14 @@ When the CB-Geo mpm code is compiled with VTK libraries, the MPM code can be set
 ```JSON
   "post_processing": {
     "output_steps": 5,
+    "path": "results/",
+    "vtk" : ["stresses","velocities"],
```

Comment:
Could you add this line to https://mpm.cb-geo.com/#/user/preprocess/input as well?

Reply:
Done!